stl/inc/cmath

```diff
@@ -953,261 +952,373 @@ _NODISCARD _CRT_SATELLITE_2 float __stdcall __std_smf_hypot3f(float, float, floa
+        else                                                                                                           \
+            return static_cast<_Common>(                                                                               \
+                __std_smf_##NAME(static_cast<double>(_Arg1), static_cast<double>(_Arg2), static_cast<double>(_Arg3))); \
+    }                                                                                                                  \
+    _STD_END
+
+#define _GENERIC_MATH_SPECIAL_UINT1(NAME)                                                         \
+    _STD_BEGIN                                                                                    \
+    template <class _Ty, enable_if_t<is_arithmetic_v<_Ty>, int> = 0>                              \
+    _NODISCARD auto NAME(const unsigned int _Arg1, const _Ty _Arg2) noexcept /* strengthened */ { \
+        using _Common = conditional_t<is_integral_v<_Ty>, double, _Ty>;                           \
+        if constexpr (is_same_v<_Common, float>)                                                  \
+            return __std_smf_##NAME##f(_Arg1, _Arg2);                                             \
+        else                                                                                      \
+            return static_cast<_Common>(__std_smf_##NAME(_Arg1, static_cast<double>(_Arg2)));     \
+    }                                                                                             \
+    _STD_END
+
+#define _GENERIC_MATH_SPECIAL_UINT2(NAME)                                                                  \
+    _STD_BEGIN                                                                                             \
+    template <class _Ty, enable_if_t<is_arithmetic_v<_Ty>, int> = 0>                                       \
+    _NODISCARD auto NAME(                                                                                  \
+        const unsigned int _Arg1, const unsigned int _Arg2, const _Ty _Arg3) noexcept /* strengthened */ { \
+        using _Common = conditional_t<is_integral_v<_Ty>, double, _Ty>;                                    \
+        if constexpr (is_same_v<_Common, float>)                                                           \
+            return __std_smf_##NAME##f(_Arg1, _Arg2, _Arg3);                                               \
+        else                                                                                               \
+            return static_cast<_Common>(__std_smf_##NAME(_Arg1, _Arg2, static_cast<double>(_Arg3)));       \
+    }                                                                                                      \
+    _STD_END
+
+// TRANSITION, VSO-945789, LWG-3234, P1467R9, mathematical special functions should have additional overloads.
+// LWG-3693, no separated float/long double overloads are provided.

```

Comment:
This PR resolves VSO-945789/GH-910; they should be closed rather than commented. This change meets the intent of LWG-3234, so it shouldn't be listed with `TRANSITION` which indicates that some additional change is necessary to agree with the issue. I'm not sure why P1467R9 is here since it hasn't been merged and we support no extended floating-point types?

I _think_ the only comment we need here is:
```suggestion
// per LWG-3234 and LWG-3693
```
but please explain if I'm missing something.